Microsoft.Toolkit.Uwp.SampleApp/SamplePages/RadialProgressBar/RadialProgressBarCode.bind

```diff
@@ -0,0 +1,28 @@
+<Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.RadialProgressBarPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+
+      <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+          <Grid VerticalAlignment="Center"
+                HorizontalAlignment="Center">
+              <Grid.ColumnDefinitions>
+                  <ColumnDefinition Width="50"></ColumnDefinition>
+                  <ColumnDefinition Width="250"></ColumnDefinition>
+                  <ColumnDefinition Width="50"></ColumnDefinition>
+              </Grid.ColumnDefinitions>
+              <controls:RadialProgressBar
+                  x:Name="RadialProgressBar"
+                  Grid.Column="1"
+                  Value="@[Value:Slider:0:0-100]"
+                  Thickness="@[Thickness:Slider:4:4-20]"
+                  Minimum="0"
+                  Maximum="100"
+                  Width="@[Width:Slider:100:100-200]"
+                  Height="@[Height:Slider:100:100-200]"/>
+          </Grid>
```

Comment:
Would it make sense to also add the Color enum for the Outline(Background) and Foreground so the show up as properties?